tests/test_appliancesmanager.py

```diff
@@ -45,51 +34,24 @@ def account_id_calls_fixture(aioresponses_mock, backend_selector_mock):
     aioresponses_mock.get(
         backend_selector_mock.shared_appliances_url, payload=shared_appliance_data
     )
-    yield
 
 
-async def test_uer_details_not_called_if_account_id_is_known(
-    auth_mock: Auth,
+async def test_get_owned_appliances_call_does_not_contain_wp_client_brand_header(
     backend_selector_mock: BackendSelector,
     appliances_manager: AppliancesManager,
     aioresponses_mock,
 ):
-    auth_mock._auth_dict["accountId"] = ACCOUNT_ID
-
     await appliances_manager.fetch_appliances()
 
-    assert (
-        "GET",
-        URL(backend_selector_mock.user_details_url),
-    ) not in aioresponses_mock.requests
+    headers = appliances_manager._create_headers()
```

Comment:
lets hard-code the headers here. that way we avoid accessing internal class stuff, and it also serves as a way to validate the headers and you can then remove the assert on line 54.

I don't really mind having some duplication or hardcoded strings on tests if they are used to validate the lib's behavior